frontend/src/pages/answer/Answer.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useContext } from 'react';
```

Comment:
Opened #192. This would definitely be nice to have. The graphs grab their size on initial render, so if you resize and then refresh the screen, they'll fit again.